index.html

```diff
@@ -551,30 +551,295 @@ <h2>SPECIAL OFFER</h2>
           <div>
           </div>
         </div>
-      </section>
-    </section>
-    <section>
-      <section class="wrapper">
-        <h2>OUR PRODUCTS</h2>
-      <ul>
-        <li><a href="#">All Products</a></li>
-        <li><a href="#">Best Sellers</a></li>
-        <li><a href="#">New Arrivals</a></li>
-        <li><a href="#">Todays Deals</a></li>
-      </ul>
-      </section>
+      </div>
+    </section> -->
+    <section class="products">
```

Comment:
здесь не хватает обозначения блока (то есть, должно быть так): 
```suggestion
    <section class="products trending">
```

запомни, нельзя использовать элементы вне блока. То есть элементы блока должны обязательно лежать внутри элемента с классом названия блока. То есть, например, элемент с классом `.trending_item` обязательно должен лежать **внутри** элемента с названием `.trending`